README.md

```diff
@@ -4,10 +4,11 @@
 The objective of this project is to create a Q&A website with a rating system on answers.
 
 ## **Description:**
-This project consists of building a Q&A website where people have the opportunity to ask and answer questions regarding a certain topic. The website will intially have three main functions:  
-1. To allow users to post questions.  
-2. To allow users to answer questions.  
-3. To allow users to vote on the answers to make more helpful answers have higher priority.  
+This project consists of building a Q&A website where people have the opportunity to ask and answer questions regarding a certain topic. The website will intially have three main features:  
+
+1. Asking and answering questions  
+2. Users voting on the answers  
+3. Poster accepting the best answer   
```

Comment:
please change n.3 to "Accepting the best answer"

Reply:
Not exaclty it is ;
1. Asking and answering questions
2. Voting on the answers
3. Accepting the best answer